### PR TITLE
fix: `LivechatSessionTaken` webhook event called without `agent` param

### PR DESCRIPTION
**Pull request by KevLehman**
## Proposed changes (including videos or screenshots)

Basically, the call to `assignAgent` was updating the room object as expected, but when calling `afterTakeInquiry` callback we were passing the room _before_ it was updated.
## Issue(s)

https://rocketchat.atlassian.net/browse/SUP-654
## Further comments

I'm planning to spin up a `webhook.test` docker image to test this one but probably in the future 😬

### CODE REVIEW

**Review by MarcosSpessatto — APPROVED** [review]

File: apps/meteor/app/livechat/server/lib/RoutingManager.ts [hunk 1]
```diff
@@ -265,11 +265,21 @@ export const RoutingManager: Routing = {
...
+			throw new Error('error-room-not-found');
+		}
+
 		callbacks.runAsync(
 			'livechat.afterTakeInquiry',
 			{
-				inquiry: await this.assignAgent(inquiry as InquiryWithAgentInfo, room, agent),
-				room,
+				inquiry: inq,
+				// @eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```
Comment (ggazzo) [inline apps/meteor/app/livechat/server/lib/RoutingManager.ts:281]: nop

**Review by ggazzo — CHANGES_REQUESTED** [review]

File: apps/meteor/app/livechat/server/lib/RoutingManager.ts (hunk 1 above)
Comment (ggazzo) [reply apps/meteor/app/livechat/server/lib/RoutingManager.ts:281]: remove the eslint/ts disable

Comment (KevLehman) [reply apps/meteor/app/livechat/server/lib/RoutingManager.ts:281]: ah, remanent of first fix.

**Review by MarcosSpessatto — CHANGES_REQUESTED** [review]
~~Is there any way of adding tests?~~ Just saw the description

**Review by MarcosSpessatto — APPROVED** [review]

**Comment by sophia-devs** [comment]
Can we get an update on this?

**Review by MartinSchoeler — APPROVED** [review]

**Pull request merged**